### PR TITLE
Fix build issue

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , Please review

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,5 @@
 NAME=tendrl-commons
-VERSION := $(shell PYTHONPATH=. python -c \
-             'import tendrl.commons; print tendrl.commons.__version__' \
-             | sed 's/\.dev[0-9]*//')
+VERSION := 1.2.3
```
Comment (r0h4n) [inline Makefile]: This seems like an exception where we are hard coding the version

Why does the older makefile work for other repos? https://github.com/Tendrl/ceph-integration/blob/develop/Makefile

Can you remove __version__ from tendrl/commons/__init__.py and make it as it is for other repos via version.py ?

Comment (TimothyAsirJeyasing) [inline Makefile]: Done

**Comment by TimothyAsir** [comment]
@r0h4n , @nthomas-redhat , @shtripat , Please review

**Comment by TimothyAsir** [comment]
@r0h4n please review
I build it locally and able to install

**Comment by r0h4n** [comment]
I am going to merge this one and fix the review comment, need this for release/1.2.3

**Pull request merged**